### PR TITLE
New pacman cheat sheet

### PR DESCRIPTION
**Pull request by abadojack**
**What does your Instant Answer do?**
My instant answer provides the most commonly used commands of the pacman package manager for Arch Linux.

**What is the data source for your Instant Answer? (Provide a link if possible)**
[Arch Wiki](https://wiki.archlinux.org/index.php/Pacman)

**Why did you choose this data source?**
Arch Wiki has the most comprehensive documentation of the pacman package manager

**Are there any other alternative (better) data sources?**
[pacman manual page](https://www.archlinux.org/pacman/pacman.8.html)

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Arch Linux users

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Are you having any problems? Do you need our help with anything?**
Yes. 

**Where did you hear about DuckDuckHack? (For first time contributors)**
NA

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
NA

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/pacman.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
+{
+    "id": "pacman_cheat_sheet",
+    "name": "Pacman",
+    "description": "Package manager for Arch Linux",
+    "metadata": {
+        "sourceName": "ArchWiki",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/pacman.json]: @abadojack "Arch Linux" would be better as the source name

File: share/goodie/cheat_sheets/json/pacman.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    },
+    "template_type": "terminal",
+    "section_order": [
+        "Installing specific packages",
+        "Installing package groups",
+        "Removing packages",
+        "Upgrading packages",
+        "Querying package databases",
+        "Cleaning package cache",
+        "Additional commands"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/pacman.json]: @abadojack Lets capitalise first letter of each word in section names

**Comment by abadojack** [comment]
Why is the The Travis CI build failing ?

**Comment by jophab** [comment]
@abadojack Sorry that I forgot to mention this before.
Shall we add some aliases,if there are?

**Comment by moollaza** [comment]
@abadojack @jophab this is failing because the Cheat Sheet already exists under a different name: 
https://duckduckgo.com/?q=pacman+cheat+sheet&t=ffab&ia=cheatsheet

@abadojack please checkout that Cheat Sheet and submit a PR that updates it, if you feel any changes are necessary.

**Comment by moollaza** [comment]
@abadojack by the way, it doesn't look like you created an Instant Answer page for this (which is alright seeing as it already exists). Jut to let you know, we've improved and simplified the process. We also have brand new documentation which explains everything: http://docs.duckduckhack.com/submitting/pull-request.html

**Pull request closed without merging**